plugin/configpulse.vim

```diff
@@ -0,0 +1 @@
+command! ConfigPulse lua require"configpulse".find_time() 
```

Comment:
This file gets loaded automatically by neovim, so you don't have to `require('configpulse')` first to get the command to register. I can remove if you don't want it though